### DISCUSSION TITLE
percentage or reads and ASVs in each taxonomic level

### DISCUSSION THREAD

**Discussion by jamorillo** [ChiLiubio/microeco#441] [Ideas]
Dear Chi,
Please excuse me if this question has been asked before. Would it be possible to know the percentage of reads (and, if possible, ASVs) assigned to each taxonomic group in a microeco object? For example: Phylum (98%), Class (70%), Order (64%), etc. This would be extremely useful when writing a paper. 
Thank you very much for all your work!
Regards,
jose

**Comment by ChiLiubio** [comment]
Hi. The percentage of reads should be the relative abundance we generally used. It is in the `taxa_abund` list generated by the function `cal_abund`.
```
library(microeco)
data(dataset)
dataset$cal_abund()
dataset$taxa_abund$Phylum
```
The percentage of ASVs is easier to get. We can directly summarize the `tax_table` like this. 
```
sort(table(dataset$tax_table$Phylum))
```

Best,
Chi
